src/pudl/analysis/mcoe.py

```diff
@@ -1,9 +1,23 @@
 """A module with functions to aid generating MCOE."""
+from typing import Any
+
 import pandas as pd
 
 import pudl
 from pudl.metadata.fields import apply_pudl_dtypes
 
+DEFAULT_GENS_COLS = [
+    "plant_id_eia",
+    "generator_id",
+    "report_date",
+    "unit_id_pudl",
+    "plant_id_pudl",
+    "plant_name_eia",
+    "utility_id_eia",
+    "utility_id_pudl",
+    "utility_name_eia",
+]
```

Comment:
This used to be a variable in the `mcoe` function. I decided to make it a global because it feels like there's really no good place for it and I want it to be more accessible / viewable from outside the `mcoe` function. I tried to just do away with default generator columns as a whole but these ones still feel pretty essential, especially since `plant_id_eia`, `generator_id` , and `report_date` are all necessary for the merge.

Reply:
I think this a good overall solution for this!